Languages/IronPython/Tests/test_privateBinding.py

```diff
@@ -129,6 +129,6 @@ def test_misc_coverage():
 
 run_test(__name__, noOutputPlease=True)
 
-if not privateBinding and not is_silverlight:
+if not privateBinding and not is_silverlight and not is_netstandard: # sys.executable isn't an executable
```

Comment:
It's a DLL. There are two types of .NET Core apps: self-contained and framework-dependant (see https://docs.microsoft.com/en-us/dotnet/articles/core/deploying/index). Self-contained produces an executable while framework-dependant produces a DLL that you run with the framework (e.g. `dotnet bla.dll`).